### DISCUSSION TITLE
Interactive user location map

### DISCUSSION THREAD

**Discussion by GWPvGWP** [macedonga/beunblurred#76] [Ideas]
A map where all shared post locations of the day are pinned. Markers could have the profile pic and after click show name and time of post.

Also it could be fun to save the locations in the archiver to make a 'user travel story map'

**Comment by macedonga** [comment]
> A map where all shared post locations of the day are pinned. Markers could have the profile pic and after click show name and time of post.

Cool idea!

> Also it could be fun to save the locations in the archiver to make a 'user travel story map'

Looks a little bit stalker-ish

**Reply by GWPvGWP** [reply]
> Looks a little bit stalker-ish

True, I also thought of that. On the other hand, it is voluntary shared by the users. But I agree it is a kind of privacy issue.
